code/modules/projectiles/updated_projectiles/magazines/pistols.dm

```diff
@@ -131,7 +131,7 @@
 	name = "\improper Highpower magazine (9mm)"
 	default_ammo = /datum/ammo/bullet/pistol/ap
 	caliber = "9mm"
-	icon_state = "m4a3" //PLACEHOLDER
+	icon_state = "m4a3_old" //PLACEHOLDER
```

Comment:
same

Reply:
That's still a placeholder kinda, it may be different now but it's still just a outdated sprite and it'd be better to have a custom one.